scripts/install.js

```diff
@@ -65,6 +66,39 @@ function createProgressBar(name, total) {
   return { tick: () => {} };
 }
 
+function copyFileSync(src, dest) {
+  const data = fs.readFileSync(src);
+  fs.writeFileSync(dest, data);
+}
+
+function npmCache() {
+  const env = process.env;
+  return (
+    env.npm_config_cache ||
+    env.npm_config_yarn_offline_mirror ||
+    (env.APPDATA ? path.join(env.APPDATA, 'npm-cache') : path.join(os.homedir(), '.npm'))
+  );
+}
+
+function getCachedPath(url) {
+  const digest = crypto
+    .createHash('md5')
+    .update(url)
+    .digest('hex')
+    .slice(0, 6);
+
+  return path.join(
+    npmCache(),
+    `${digest}-${path.basename(url).replace(/[^a-zA-Z0-9.]+/g, '-')}`
```

Comment:
What do you think about adding an additional intermediate directory (e.g. `sentry-cli`) to place the binaries there?

Reply:
yep, good idea!